packages/block-editor/src/components/block-list/style.scss

```diff
@@ -737,6 +737,7 @@
 	left: 0;
 	right: 0;
 	justify-content: center;
+	height: $block-padding + 8px;
 
 	// Show a clickable plus.
 	.block-editor-inserter__toggle {
```

Comment:
Below this line, it says `margin-top: -4px`. If you change that to -6px, it better vertically centers the plus between the blocks. This was not a regression of this PR, but would be nice to fix while you're in here.

Reply:
@jasmussen yes that sounds good. Would it be a good idea to also add a comment there to indicate that this is indeed a stopgap fix? So that when the real fix does come along these changes are reverted.

Reply:
Maybe. Or maybe we grow to like the change enough to keep it, even if we also fix the other issue.

I can go either way.

Reply:
Sounds good. I've made the change to centre align it. I went with -8px instead of -6px as it accurately centre aligns it.